operators/community-windows-machine-config-operator/3.0.0/manifests/community-windows-machine-config-operator.3.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,320 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Seamless Upgrades
+    categories: "OpenShift Optional"
+    certified: "false"
+    containerImage: quay.io/openshift-windows/community-windows-machine-config-operator:community-4.8-5a0c01f
+    createdAt: 2021-08-05T18:20:00Z
+    description: An operator that enables Windows container workloads on OKD/OCP
+    operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
+    operators.operatorframework.io/builder: operator-sdk-v2.0.0+git
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    repository: https://github.com/openshift/windows-machine-config-operator
+    support: ""
+    olm.skipRange: ">=2.0.0 <3.0.0"
+  name: community-windows-machine-config-operator.v3.0.0
+  namespace: openshift-windows-machine-config-operator
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned: []
+  description: |-
+    ## WARNING
+    Community distribution of the Windows Machine Config Operator.
+
+    This is an preview build, and is not meant for production workloads.
```

Comment:
Looks great! Only thing I noticed:
"This is an preview build" change to----> "This is a preview build"